tools/release/release-output/release-packages.ts

```diff
@@ -8,3 +8,8 @@ export const releasePackages = [
   'material-experimental',
   'material-moment-adapter',
 ];
+
+export const packagesWithMigrations = [
```

Comment:
can we just check `packageJson['ng-update']` to determine this automatically without needing to maintain a separate list?

Reply:
Done.